.dockerignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
You're being careful about what you copy in so you don't need much in this file, but one thing that you could add is `**/__pycache__`